content/docs/overview/observer/index.md

```diff
@@ -1,8 +1,8 @@
 ---
-contributors: {'Balázs Kreith', 'Balazs Kreith'}
-title: "Observer"
+contributors: {'Balázs Kreith', 'Chad Hart'}
```

Comment:
This page is generated by a script.
The original page is here: https://github.com/ObserveRTC/observer/blob/master/docs/user-manual.md 
if you modify the source, the script automatically fetch your name as author and the generated page will contain it.